<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GotoDialog</class>
 <widget class="QDialog" name="GotoDialog">
  <property name="windowTitle">
   <string>Coordinates</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rc/icons.qrc">
    <normaloff>:/icons/logo.svg</normaloff>:/icons/logo.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Coordinates:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelX">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineX"/>
      </item>
      <item>
       <widget class="QLabel" name="labelZ">
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineZ"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Scale:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineScale"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>blocks per pixel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rc/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GotoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GotoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
